### PR TITLE
Functional tests - Separate create date from test to Utils

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Separate create date from test to common tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | o
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26815)

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js [hunk 1]
```diff
@@ -593,7 +588,7 @@ describe('BO - Orders - View and edit order : Check payment Block', async () =>
     it('should add payment when amount is equal to the rest to the new invoice', async function () {
       await testContext.addContextItem(this, 'testIdentifier', 'testAmountSupTotal', baseContext);
 
-      const invoice = `#IN0000${invoiceID >= 10 ? '' : '0'}${invoiceID + 1}`;
+      const invoice = `#IN0000${invoiceID + 1 >= 10 ? '' : '0'}${invoiceID + 1}`;
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js]:
```suggestion
      const invoice = `#IN0000${(invoiceID + 1) >= 10 ? '' : '0'}${invoiceID + 1}`;
```

**Review by Progi1984 — COMMENTED** [review]
@nesrineabdmouleh A small feedback

**Review by PierreRambaud — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js (hunk 1 above)
Comment (PierreRambaud) [reply tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js]: Not mandatory, bring no value to add parenthesis :sweat_smile:

**Review by NeOMakinG — DISMISSED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh & @boubkerbribri

**Pull request merged**